diff --git a/libgnome/gnome-config.h b/libgnome/gnome-config.h
index 0c3dfe6..b9d659d 100644
--- a/libgnome/gnome-config.h
+++ b/libgnome/gnome-config.h
@@ -270,7 +270,7 @@ void gnome_config_clean_key_ (const char *path, gboolean priv);
 #define gnome_config_private_clean_key(path) \
 	(gnome_config_clean_key_((path),TRUE))
 
-/* returns the true filename of the config file */
+/* returns the true filename of the config file */
 #define gnome_config_get_real_path(path) \
 	(g_build_filename (gnome_user_dir_get(),(path),NULL))
 #define gnome_config_private_get_real_path(path) \
